src/services/extensions/links.ts

```diff
@@ -120,3 +133,26 @@ export function appendUrlParameter(
 export function replaceSlash(parameter: string): string {
   return parameter.replace(/\//g, '-');
 }
+
+// Manually copied over from @grafana/scenes so we don't need to import scenes to build links
```

Comment:
Are these covered by unit tests? Maybe indirectly in links.test.ts, but just double checking.